src/App.jsx

```diff
@@ -39,28 +39,28 @@ class App extends React.Component {
                     <Switch>
-                        <Route exact path="/login" key="Login" render={ props => <CustomAuthenticator onSignIn={this.signInHandler}/> } />
+                        <Route exact path="/inloggen" key="Login" render={ props => <CustomAuthenticator onSignIn={this.signInHandler}/> } />
                         {/* <Route exact path="/verify?type=:verificationType&user=:username&code=:verificationCode" key="Verify" render={ props => <CustomAuthenticator onSignIn={this.signInHandler}/> } /> */}
                         {/* <Route exact path="/verify" key="Verify" render={ props => <CustomAuthenticator onSignIn={this.signInHandler}/> } /> */}
 
-                        <Route exact path="/about" key="Who are we" component={About}/>
+                        <Route exact path="/over-ons" key="Who are we" component={About}/>
                         <Route exact path="/contact" key="Contact us" component={Contact}/>
-                        <Route exact path="/privacy" key="Privacy Statement" component={Privacy}/>
-                        <Route exact path="/faq" key="Frequently Asked Questions" component={FAQ}/>
-                        <Route exact path="/terms" key="Terms of Use" component={Terms}/>
+                        <Route exact path="/privacyverklaring" key="Privacy Statement" component={Privacy}/>
+                        <Route exact path="/veelgestelde-vragen" key="Frequently Asked Questions" component={FAQ}/>
+                        <Route exact path="/gebruiksvoorwaarden" key="Terms of Use" component={Terms}/>
 
-                        <Route exact path="/workspace/" key="Workspace" component={Workspace}/>
+                        <Route exact path="/actie/" key="Workspace" component={Workspace}/>
 
-                        {/* <Route exact path="/workspace/:initiativeId" key="WorkspacePage" component={Workspace}/>
-                        <Route exact path="/workspace/:initiativeId/login" key="WorkspaceLogin" render={ props => <Workspace {...props} signInHandler={this.signInHandler}/> } /> */}
+                        {/* <Route exact path="/actie/:initiativeName" key="WorkspacePage" component={Workspace}/>
+                        <Route exact path="/actie/:initiativeName/inloggen" key="WorkspaceLogin" render={ props => <Workspace {...props} signInHandler={this.signInHandler}/> } /> */}
                         
-                        <Route exact path="/workspace/login" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
-                        <Route exact path="/workspace/add-find-playground" key="WorkspaceAddFindPlayground" component={Workspace}/>
+                        <Route exact path="/actie/inloggen" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
+                        <Route exact path="/actie/zoeken" key="WorkspaceAddFindPlayground" component={Workspace}/>
```

Comment:
After seeing this implemented, I would change url to /actie/starten instead of zoeken to align it with h1

Reply:
Done!